docs/static/releasenotes.asciidoc

```diff
@@ -43,6 +43,22 @@ https://github.com/elastic/logstash/pull/11772[#11772]
 * Fixed: when x-pack Monitoring is configured with `cloud_id`, the monitoring pipeline now correctly resolves the hosts.
 https://github.com/elastic/logstash/pull/11800[#11800]
 
+==== Known issue
+
```

Comment:
I'd create a subtitle here that reads something like
Issue https://github.com/elastic/logstash/issues/12031[#12031]: Pipeline compilation may increase start-up time significantly.

A potential performance regression [...]